### PR TITLE
IDEM-363

### PR DESCRIPTION
**Pull request by JasminFarhi**
Screen Shot 2022-08-16 at 2 42 04 pm

### CODE REVIEW

**Review by xiangzhe0812 — DISMISSED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -79,6 +69,23 @@ export const App = () => {
...
+              IDEM is a fully open source application shipped under MIT licence.
+              Feel free to raise an issue on
+              <a
+                className="flex text-[#ffd000] text-xl mb-10 w-20% justify-center px-1"
+                href="https://github.com/dltxio/idem-mobile"
+              >
+                GitHub.
+              </a>
+              <div className="flex text-white text-xl mb-10 w-20% justify-center">
+                Or conact support below.
```
Comment (Mitchell139) [inline src/App.tsx]:
```suggestion
                Or contact support below.
```

File: src/App.tsx [hunk 2]
```diff
@@ -79,6 +69,23 @@ export const App = () => {
...
+              </div>
+            </div>
+
+            <div className="flex justify-center text-center pb-10">
+              <Form />
```
Comment (Mitchell139) [inline src/App.tsx]:
```suggestion
```

File: src/components/Form.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const RedeemCodeForm = () => {
```
Comment (Mitchell139) [inline src/components/Form.tsx]: What's redeem code?

**Review by Mitchell139 — CHANGES_REQUESTED** [review]

**Review by Mitchell139 — APPROVED** [review]

**Review by xiangzhe0812 — APPROVED** [review]

File: src/App.tsx [hunk 4]
```diff
@@ -5,6 +5,7 @@ import About from "./pages/About";
 import Background from "./assets/Background.svg";
 import testGroupQRCode from "./assets/testGroupQRCode.png";
 import IdemVideo from "./components/IdemPromoVideo";
+import ContactLink from "./components/Form";
```
Comment (brettvniekerk) [inline src/App.tsx:8]: Add IdemVideo and ContactLink to line 1

File: src/App.tsx [hunk 5]
```diff
@@ -79,6 +69,23 @@ export const App = () => {
                 <IdemVideo />
               </div>
             </div>
+            <div className="flex text-white text-xl mb-10 w-20% justify-center">
```
Comment (brettvniekerk) [inline src/App.tsx:72]: `w-[20%]` no? otherwise use `w-1/5`

File: src/App.tsx [hunk 6]
```diff
@@ -79,6 +69,23 @@ export const App = () => {
...
+              IDEM is a fully open source application shipped under MIT licence.
+              Feel free to raise an issue on
+              <a
+                className="flex text-[#ffd000] text-xl mb-10 w-20% justify-center px-1"
```
Comment (brettvniekerk) [inline src/App.tsx:76]: Why `flex` for anchor tag? Same width question as above

File: src/App.tsx (hunk 5 above)
Comment (brettvniekerk) [reply src/App.tsx:72]: Also why flex

File: src/App.tsx [hunk 7]
```diff
@@ -79,6 +69,23 @@ export const App = () => {
...
+                href="https://github.com/dltxio/idem-mobile"
+              >
+                GitHub.
+              </a>
+              <div className="flex text-white text-xl mb-10 w-20% justify-center">
```
Comment (brettvniekerk) [inline src/App.tsx:81]: Same as above

File: src/components/Form.tsx [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <div className="w-2/5 flex flex-col items-center mt-10">
+      <h1 className="text-[#ffd000] text-5xl pt-50 w-full">Support</h1>
+      <h2 className="text-white text-xl w-full mt-2">
+        Get in contact with our team and we will get back to you shortly.
+      </h2>
+      <a
+        className=" flex bg-[#ffd000] text-black md:text-base text-center rounded-3xl my-10 py-5 px-20 hover:bg-[#ab8d09]"
+        target="_blank"
+        href="mailto:support@dltx.io"
```
Comment (brettvniekerk) [inline src/components/Form.tsx:11]: Needs `rel="noreferrer noopener"` property for security

**Review by brettvniekerk — CHANGES_REQUESTED** [review]

**Pull request merged**